DQM/SiStripMonitorHardware/src/SiStripFEDDataCheck.cc

```diff
@@ -340,7 +340,7 @@ void SiStripFEDCheckPlugin::bookHistograms(DQMStore::IBooker & ibooker , const e
 
 // ------------ method called once each run just after ending the event loop  ------------
 void 
-SiStripFEDCheckPlugin::endRun()
+SiStripFEDCheckPlugin::endRun(edm::Run const&, edm::EventSetup const&)
```

Comment:
This changes the signaure of the function so changes the behavior (since the framework will now actually call the function).